cypress/page-objects/lib/main-table-checkbox.ts

```diff
@@ -0,0 +1,23 @@
+import { CommonSelectors } from '../../../src/components/data-testid/common-selectors.cy'
+import { MainTableDataCy } from '../../../src/components/data-testid/main-table.cy'
+import { Checkbox } from './checkbox'
+
+export class MainTableCheckbox extends Checkbox {
+  constructor(selector: string) {
+    super(selector)
+  }
+  variantMainTableCheckbox(substring?: string) {
+    cy.get(CommonSelectors.checkBoxList)
+      .find(`[data-testid = "${MainTableDataCy.checkboxListElement}"]`)
+      .contains(substring!)
+      .siblings(CommonSelectors.checkbox)
+      .click({ force: true })
+  }
+  checkTagInDrawer(substring?: string, time?: number) {
+    cy.get(CommonSelectors.checkBoxListInDrawer, { timeout: time })
```

Comment:
It's ok to use the same parameter name, it could be `{ timeout: timeout }` and in this case you can just use `{ timeout }`, it will be the same. Please change time => timeout.